Automattic/wp-calypso
# Merge eslint-plugin-wpcalypso and eslint-config-wpcalypso

#### Changes proposed in this Pull Request

Comes from #32299

* Merges `eslint-config-wpcalypso` and `eslint-plugin-wpcalypso` into a single package.
* Clean up duplicated plugin imports in `.eslintrc.js`

#### Testing instructions

* Check that `Code Style` build reports the same errors in this branch than `trunk`.
